src/store/actions/actions.js

```diff
@@ -1,42 +1,55 @@
 import { API } from '../../middleware/apiMiddleware';
 import config from '../../config/config';
 
-const fetchUserAction = {
-  type: 'FETCH_USER',
+//CATEGORIES
+const fetchCategoriesAction = {
+  type:'FETCH_CATEGORIES',
   [API]: {
-    headers: {
-      'Authentication': 'lsakjdvoa97va3s2', //FacebookUserToken,
-      'Content-Type': 'Application/JSON',
-    },
-    endpoint: '/me',
+    endpoint: '/categories',
   }
 }
 
-const fetchUserLocationAction = {
-  type: 'USER_LOCATION',
+//CONVERSATIONS
+const fetchConversationsActionCreator = (userToken) => ({
+  type:'FETCH_CONVERSATIONS',
   [API]: {
-    externalUrl: config.geolookup_url,
+    headers: {
+      'Authentication': userToken,
+      'Content-Type': 'Application/JSON',
+    },
+    endpoint: '/conversation',
   }
-}
+});
 
-const fetchCategoriesAction = {
-  type:'GET_CATEGORIES',
+const fetchProfileActionCreator = (userToken) => ({
+  type: 'FETCH_PROFILE',
   [API]: {
-    endpoint: '/categories',
+    headers: {
+      'Authentication': userToken,
```

Comment:
The correct header is `Authorization` and it should follow a stratgy (`Bearer` in this case)